docs/cellxgene_census_docsite_landing.md

```diff
@@ -1,8 +1,6 @@
-<span style="background-color: #f3bfcb; color; font-size: 18px"> 🚀 Now in testing: Spatial! From Jan 16th, latest builds will include data from Slide-seq and Visium assays. ⚠️ Opening these builds requires `tiledbsoma>=1.15.3` ⚠️. [Learn more](articles/2025/20250117-spatial.md)!
+<span style="background-color: #f3bfcb; color; font-size: 18px"> 🚀 Census LTS release `2025-11-08` includes CELLxGENE Discover datasets from _Macaca mulatta_, _Callithrix jacchus_, and _Pan troglodytes_ as well as _Homo sapiens_ and _Mus musculus_.</span>
```

Comment:
For my own education, for a while Census was always a subset of the full CxG corpus, namely only having human and mouse datasets. With this new version, is it still the case that Census is still a subset of the full CxG corpus?

Thank you!